is_core/generic_views/form_views.py

```diff
@@ -353,6 +353,9 @@ def get_form_class(self):
     def get_form_class_base(self):
         return self.form_class or SmartModelForm
 
+    def get_is_bulk(self):
```

Comment:
nebyla by lepší property, kterou tu použiješ? dá se to pak přetížit a snadněji změnit

Reply:
Jako asi jo, ale teď už je merged. Pokud chceš můžeš to změnit ;-)